packages/observable-md/src/util.ts

```diff
@@ -54,3 +65,33 @@ export function encodeMD(str: string) {
         .split("$").join("\\$")
         ;
 }
+
+export function encodeBacktick(str: string) {
+    return str
+        .split("`").join("\\`")
```

Comment:
Why two backslashes here but one backslash on line 57?

Reply:
Its needed to make the following (broken code) work:
``````javascript
const innerStr = "`x`";
eval(`md\`${x}\`);
``````

Reply:
Compared to:
```
const innerStr = "`x`";
eval(`${x}`);
```